server/events/yaml/raw/repo_cfg.go

```diff
@@ -10,12 +10,16 @@ import (
 // DefaultAutomerge is the default setting for automerge.
 const DefaultAutomerge = false
 
+// DefaultParallelPlans is the default setting for parallel plans
+const DefaultParallelPlans = false
```

Comment:
Is there any reason you wouldn't want this automatically turned on? Wondering if we need to expose this in config at all.

Reply:
I originally had this as an option because some of our repos weren't using workspaces. I like the idea of always having it turned on, I can change it so it detects if workspaces are being used or not

Reply:
made this default now for plans, but added a flag for apply